### PR TITLE
docs: Suppress Mkdocs warnings

### PR DESCRIPTION
**Pull request by mAxYoLo01**
## Pull Request Type

- [ ] Feature addition
- [ ] Bugfix
- [x] Documentation update
- [ ] Code refactor
- [ ] Tests improvement
- [ ] CI/CD pipeline enhancement
- [ ] Other: [Replace with a description]

## Description

This PR will focus solely on docs.
For the moment, it is kept as a draft until I find how to resolve a few more issues I could see that would make our current docs better.

## Python Compatibility

- [x] I've ensured my code works on Python `3.10.x`
- [x] I've ensured my code works on Python `3.11.x`

## Checklist

- [x] I've run the `pre-commit` code linter over all edited files
- [x] I've tested my changes on supported Python versions
- [ ] I've added tests for my code, if applicable
- [x] I've updated / added  documentation, where applicable

### CODE REVIEW

**Comment by mAxYoLo01** [comment]
Wow just saw that my isort is not happy with the repo lol

File: interactions/ext/prefixed_commands/context.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ async def reply(
         content: Optional[str] = None,
         embeds: Optional[Union[Iterable[Union[Embed, dict]], Union[Embed, dict]]] = None,
         embed: Optional[Union[Embed, dict]] = None,
-        **kwargs,
+        **kwargs: dict,
```
Comment (silasary) [inline interactions/ext/prefixed_commands/context.py]: This is not correct.
`**` typehints are implicitly `dict[str, T]` where T is your provided hint.  You're saying "Any additional args must be dicts", where I'd normally expect **kwargs to be hinted `Any`

**Review by silasary — CHANGES_REQUESTED** [review]

File: interactions/ext/prefixed_commands/context.py (hunk 1 above)
Comment (mAxYoLo01) [reply interactions/ext/prefixed_commands/context.py]: In the libs code, I found there are probably 10 different ways kwargs are typehinted, the simplest one was just a basic dict so I went with that to please MkDocs

**Comment by mAxYoLo01** [comment]
PR ready, I forgot it existed lol

File: interactions/ext/prefixed_commands/context.py (hunk 1 above)
Comment (silasary) [reply interactions/ext/prefixed_commands/context.py]: You're misunderstanding [PEP 484](https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values) here.  
This is what you've annotated the parameter as: 
image

Yes, there are some other places in the library that also do it incorrectly.  (And also some places where the typing is in fact [Dict[str, Dict[str, Any]]](https://github.com/interactions-py/interactions.py/blob/stable/interactions/api/http/http_client.py#L350)) 

You can fix them, or you can not.  But I do not want to merge a PR that makes it worse.

Comment (mAxYoLo01) [reply interactions/ext/prefixed_commands/context.py]: Ok I indeed was misunderstanding how it typehinted kwargs, thanks for that info, I have modified it.

**Review by silasary — APPROVED** [review]

**Pull request merged**